### PR TITLE
bug(category): fix category bug where magento price filters with an *…

### PR DESCRIPTION
**Pull request by lderrickable**
… were applied incorrectly

## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/graycoreio/daffodil/blob/master/CONTRIBUTING.md#commit
- [x] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?
```
[x] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Build related changes
[ ] CI related changes
[ ] Documentation content changes
[ ] Other... Please describe:
```

## Does this PR introduce a breaking change?
```
[ ] Yes
[x] No
```

### CODE REVIEW

File: libs/category/src/drivers/magento/transformers/applied-filter-transformer.service.spec.ts [hunk 1]
```diff
@@ -57,6 +57,26 @@ describe('DaffMagentoAppliedFiltersTransformService', () => {
 
 				expect(service.transform(categoryId, categoryFilterActions)).toEqual(expectedReturn);
 			});
+
+			it('should transform a format of ##-* into a valid magento FromTo filter', () => {
```
Comment (damienwebdev) [inline libs/category/src/drivers/magento/transformers/applied-filter-transformer.service.spec.ts]: What about ##-*?

Comment (lderrickable) [reply libs/category/src/drivers/magento/transformers/applied-filter-transformer.service.spec.ts]: I mean like 40-*, 50-*, etc.

Comment (damienwebdev) [reply libs/category/src/drivers/magento/transformers/applied-filter-transformer.service.spec.ts]: Oh, sorry. I meant the other side....

```
##-*
vs.
*-##
```

Comment (lderrickable) [reply libs/category/src/drivers/magento/transformers/applied-filter-transformer.service.spec.ts]: done

**Pull request merged**